<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather Visualizing Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>621</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlot">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>161</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="stateLabel">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stateBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>191</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cityBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonFit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit Linear Model</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="regressionOutputBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Regression results:</string>
    </property>
    <widget class="QLabel" name="labelEquation">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y = mx + b</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m: </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="labelFrom">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditFrom"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QLabel" name="labelTo">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditTo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>221</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QLabel" name="labelStartTime">
       <property name="text">
        <string>First available: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelEndTime">
       <property name="text">
        <string>Last available:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonExtremeUpdate">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>540</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>467</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Time taken (ms):</string>
    </property>
    <widget class="QLabel" name="labelTimeResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelNumberOfDays">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>468</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of days</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNumDaysExtreme">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>487</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>704</x>
      <y>375</y>
      <width>111</width>
      <height>84</height>
     </rect>
    </property>
    <property name="title">
     <string>Temperature select</string>
    </property>
    <widget class="QRadioButton" name="radioButtonHottest">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>28</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Hottest</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonColdest">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>53</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Coldest</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>832</x>
      <y>375</y>
      <width>121</width>
      <height>84</height>
     </rect>
    </property>
    <property name="title">
     <string>Sort method select</string>
    </property>
    <widget class="QRadioButton" name="radioButtonHeap">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>28</y>
       <width>50</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Heap</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMerge">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>53</y>
       <width>55</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Merge</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelExtremes">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Extreme Temperatures (Â°F)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listExtremes">
    <property name="geometry">
     <rect>
      <x>702</x>
      <y>81</y>
      <width>256</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stateBox</tabstop>
  <tabstop>cityBox</tabstop>
  <tabstop>dateEditFrom</tabstop>
  <tabstop>dateEditTo</tabstop>
  <tabstop>pushButtonPlot</tabstop>
  <tabstop>pushButtonFit</tabstop>
  <tabstop>radioButtonHottest</tabstop>
  <tabstop>radioButtonColdest</tabstop>
  <tabstop>radioButtonHeap</tabstop>
  <tabstop>radioButtonMerge</tabstop>
  <tabstop>lineEditNumDaysExtreme</tabstop>
  <tabstop>pushButtonExtremeUpdate</tabstop>
  <tabstop>listExtremes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
